locale/content.json

```diff
@@ -1221,7 +1221,7 @@
       "location": "Lleoliad",
       "provideInformationOnline": "Darparu eich gwybodaeth ar-lein",
       "uploadLettersDocumentPhotos": "Uwchlwytho llythyrau, dogfennau neu luniau",
-      "uploadLettersDocumentPhotosAudioVideo": "Uwchlwytho llythyrau, dogfennau neu ffeiliau llun, sain neu fideo",
+      "uploadAudioVideo": "Upload audio or video files",
```

Comment:
Shouldn't this be Welsh translation

Reply:
We have separate ticket (https://tools.hmcts.net/jira/browse/SSCS-9687)  for welsh translation